RELEASE_NOTES.md

```diff
@@ -187,7 +187,27 @@ For more information about this process, please refer to the documentation at ht
 
 You can see more details about this change on PR [\#13669](https://github.com/decidim/decidim/pull/13669)
 
-### 3.5. [[TITLE OF THE ACTION]]
+### 3.5. Social login changes
+
+We have changed the icons for the social logins so they align better with the social networks guidelines (Twitter/X, Facebook, and Google). If you do not use any of these social logins you can skip this step.
+
+If on the other hand you have set up this social logins, you can change it by replacing your `config/secrets.yml`:
+
+As an example, where it says:
```

Comment:
Let's add examples for FB and Twitter/X, as the svg file for twitter isn't intuitive (I tried "x.svg" and "twitter.svg" and it was "twitter-x.svg")

Reply:
Updated with paths to the other SVG's here: 1503e232857f1d6c42bc8c1356f1a39b6c325e50